client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -40,7 +40,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Do you sell domains?' ) }
 				answer={ translate(
-					'Yes! Annual and biannual Personal, Premium, Business, and eCommerce plans include a free custom domain for one year. ' +
+					'Yes! Annual and biennial Pro plan include a free custom domain for one year. ' +
```

Comment:
Should probably say `Yes! Pro plan includes a free ...` since there's only the annual Pro plan